test/bn_peract_test.cpp

```diff
@@ -233,11 +233,11 @@ struct verify_forward_train_bn_per_activation
             runMean = tensor<U>{rs_n_batch, rs_channels, rs_height, rs_width};
             runVar  = tensor<U>{rs_n_batch, rs_channels, rs_height, rs_width};
 
-            const U Data_scale = static_cast<U>(0.001);
+            const double Data_scale = 0.001;
```

Comment:
[Notice] This seems fine so far, but what about cases when U is not one of the natively supported types (e.g. BF16 or FP8)? Just curious.